assets/static/css/style.css

```diff
@@ -0,0 +1,75 @@
+* {
```

Comment:
nit: this file needs formatted

Reply:
Do we need a CSS linter?

Reply:
Seems like overkill at the moment, possibly a todo. Who knows if we will even build any more UIs?

Reply:
Idk - https://stylelint.io/ seems pretty easy to wire into pkg so we're consistent.